src/components/transactions/styles.ts

```diff
@@ -0,0 +1,53 @@
+import styled from '@emotion/styled';
+import typography from 'src/theme/typography';
+import { PRIMARY, SECONDARY } from 'src/theme/colors';
+
+export const TransactionsModalWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 20px 0px 60px 0px;
+  background-color: ${PRIMARY.white};
+  width: 600px;
+  height: 100%;
+  padding-bottom: 50px;
+`;
+
+export const TransactionModalTitle = styled.div`
+  font-weight: ${typography.fontWeightSemiBold};
+  font-size: ${typography.h3};
+  color: ${PRIMARY.black};
+  width: 90%;
+  display: flex;
+  justify-content: flex-start;
+`;
+
+export const TransactionWrapper = styled.div``;
```

Comment:
Do you need this empty div?

Reply:
and in figma top-up and payment have opposite colors

<img width="345" alt="image" src="https://github.com/ZenBit-Tech/ctrlchamps_fe/assets/117380986/205bff0a-a023-43cf-9680-5476ed8e1b1d">